catalog/schema/workflows.yaml

```diff
@@ -0,0 +1,46 @@
+id: https://github.com/galaxyproject/brc-analytics/blob/main/catalog/schema/workflows.yaml#
+name: workflows
+description: Schema for source workflows list.
+
+prefixes:
+  linkml: https://w3id.org/linkml/
+
+imports:
+  - linkml:types
+  - ./enums/workflow_type
+  - ./enums/workflow_ploidy
+
+classes:
+  Workflows:
+    description: Object containing list of workflows.
+    tree_root: true
+    attributes:
+      workflows:
+        description: List of workflows.
+        required: true
+        multivalued: true
+        range: Workflow
+
+  Workflow:
+    description: A workflow.
+    attributes:
+      trs_id:
+        description: The workflow's TRS ID.
+        required: true
+        range: string
+      type:
```

Comment:
Could this be called categories and be an array since some workflows fit in more than one category ? I'd reserve type for iterations of the workflow format itself (we currently use .ga files but plan to use gxformat2 in the future).